src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -0,0 +1,130 @@
+// This file is dual licensed under the terms of the Apache License, Version
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+use std::collections::HashSet;
+
+use once_cell::sync::Lazy;
+
+use cryptography_x509::common::{
+    AlgorithmIdentifier, AlgorithmParameters, RsaPssParameters, PSS_SHA256_HASH_ALG,
+    PSS_SHA256_MASK_GEN_ALG, PSS_SHA384_HASH_ALG, PSS_SHA384_MASK_GEN_ALG, PSS_SHA512_HASH_ALG,
+    PSS_SHA512_MASK_GEN_ALG,
+};
+
+/// Permitted algorithms, from CA/B Forum's Baseline Requirements, section 7.1.3.2 (pages 96-98)
+/// https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-v2.0.0.pdf
+pub static WEBPKI_PERMITTED_ALGORITHMS: Lazy<HashSet<AlgorithmIdentifier<'_>>> = Lazy::new(|| {
```

Comment:
How do you envision this being used?

Reply:
I envisoned it as part of the algorithms restriction functionality added with #9405: https://github.com/pyca/cryptography/pull/9405/files#diff-b15c4ba048c4bbef3e3e40afef6eaf31c7edcdd7d46627cb470c3c8db684a040R223-R254

Reply:
Ok we might need to take a step back here, because when we spoke I think we agreed to move away from the profiles idea, so I'm struggling a bit to understand how the pieces fit together here.

Reply:
Yeah, there are no more profiles in the current design -- there's only one "policy," and in our last discussion we came to the conclusion (IIRC) that the right way to offer configurability here was to allow "helper" instantiations like `Policy::webpki()` that pre-fill the things that would otherwise go under a full "profile."